### ISSUE TITLE
Commerce Cart Messages are not styled properly

### ISSUE THREAD

**Issue by jacine** [apigee/apigee-devportal-kickstart-drupal#184]
**Describe the bug**
Not sure if Commerce is hard-coding this, or if it's in a template, or potentially JS theme function, but it's using the core markup, as opposed to the overrides we have in place.

**To Reproduce**
1. Try to purchase a rate plan.
2. Go through checkout.
3. Insert a problematic credit card number.
4. See unstyled error.

**Expected behavior**
Message should look like the others.

**Screenshots**
Message is shown above credit card field.
![cart-error-messages-not-styled](https://user-images.githubusercontent.com/60979/60831081-b5d78200-a186-11e9-9111-b97010cb49a9.png)

**Comment by shadcn** [comment]
I have not been able to reproduce this. This is what I see.

![Order_information___Apigee_Inc](https://user-images.githubusercontent.com/124599/60888401-c57ac780-a267-11e9-9c04-eb19c1a5eab8.jpg)

Which payment gateway is selected?

**Comment by jacine** [comment]
Wow, ok... sorry, this is critical info that I forgot to add!  

I'm using "Authorize.net (Accept.js)" plugin via `drupal/commerce_authnet`.    And this is the markup:

```html
<div class="authorize-net-accept-js-form js-form-wrapper form-wrapper" data-drupal-selector="edit-payment-information-add-payment-method-payment-details" id="edit-payment-information-add-payment-method-payment-details--clBrgoe25as">
<div id="payment-errors">
<div class="messages messages--error">Please provide valid credit card number.</div>
</div>
...
</div>
```

Since I can easily reproduce this, I can take it ;)

**Comment by jacine** [comment]
This was fixed by #202.
